config/config.go

```diff
@@ -853,17 +853,31 @@ type OracleConfig struct {
 	PruneInterval time.Duration `mapstructure:"prune_interval"`
 	// Max allowable size for votes that can be gossiped from peer to peer
 	MaxGossipMsgSize int `mapstructure:"max_gossip_msg_size"`
+	// Enables sub account signing for votes
+	EnableSubAccountSigning bool `mapstructure:"enable_sub_account_signing"`
+	// Path to the JSON file containing the subaccount key to use to sign oracle votes
+	SubAccountKeyFilePath string `mapstructure:"sub_account_key_file_path"`
 }
 
+const (
+	DefaultOracleSubAccountKeyName = "oracle_sub_account_key.json"
```

Comment:
the legacy oracle service didnt have a key file like this because the key was added directly to the keyring, but since we are signing the oracle votes from cometbft, i used the same format as to how they import validator keys to sign block votes/proposals